packages/spark-core/utilities/_utilities.scss

```diff
@@ -2,166 +2,301 @@
+  color: $red-dark !important;
+}
+
+.sprk-u-Color--orange-medium {
+  color: $orange-medium !important;
+}
+
+.sprk-u-Color--orange-dark {
+  color: $orange-dark !important;
+}
+
+.sprk-u-Color--gray-light {
+  color: $gray-light !important;
+}
+
+.sprk-u-Color--gray-medium {
+  color: $gray-medium !important;
+}
+
+.sprk-u-Color--gray-dark {
+  color: $gray-dark !important;
+}
+
+.sprk-u-Color--black {
+  color: $black !important;
+}
+
+.sprk-u-Color--white {
+  color: $white !important;
+}
+
+// Background color
 .sprk-u-BackgroundColor--green-light {
-  background-color: $green-light;
+  background-color: $green-light !important;
 }
 
 .sprk-u-BackgroundColor--green-medium {
-  background-color: $green-medium;
+  background-color: $green-medium !important;
 }
 
 .sprk-u-BackgroundColor--green-dark {
-  background-color: $green-dark;
+  background-color: $green-dark !important;
 }
 
 .sprk-u-BackgroundColor--blue-light {
-  background-color: $blue-light;
+  background-color: $blue-light !important;
 }
 
 .sprk-u-BackgroundColor--blue-medium {
-  background-color: $blue-medium;
+  background-color: $blue-medium !important;
 }
 
 .sprk-u-BackgroundColor--blue-dark {
-  background-color: $blue-dark;
+  background-color: $blue-dark !important;
+}
+
+.sprk-u-BackgroundColor--purple-medium {
+  background-color: $purple-medium !important;
 }
 
-.sprk-u-BackgroundColor--purple {
-  background-color: $purple-medium;
+.sprk-u-BackgroundColor--purple-dark {
+  background-color: $purple-dark !important;
 }
 
 .sprk-u-BackgroundColor--red-light {
-  background-color: $red-light;
+  background-color: $red-light !important;
 }
 
 .sprk-u-BackgroundColor--red-medium {
-  background-color: $red-medium;
+  background-color: $red-medium !important;
 }
 
 .sprk-u-BackgroundColor--red-dark {
-  background-color: $red-dark;
+  background-color: $red-dark !important;
 }
 
 .sprk-u-BackgroundColor--orange-medium {
-  background-color: $orange-medium;
+  background-color: $orange-medium !important;
 }
 
 .sprk-u-BackgroundColor--orange-dark {
-  background-color: $orange-dark;
+  background-color: $orange-dark !important;
 }
 
 .sprk-u-BackgroundColor--gray-light {
-  background-color: $gray-light;
+  background-color: $gray-light !important;
 }
 
 .sprk-u-BackgroundColor--gray-medium {
-  background-color: $gray-medium;
+  background-color: $gray-medium !important;
 }
 
 .sprk-u-BackgroundColor--gray-dark {
-  background-color: $gray-dark;
+  background-color: $gray-dark !important;
 }
 
 .sprk-u-BackgroundColor--black {
-  background-color: $black;
+  background-color: $black !important;
 }
 
 .sprk-u-BackgroundColor--white {
-  background-color: $white;
+  background-color: $white !important;
+}
+
+//
+// Multi-property utilities
+///
+
+// Accessibility
+.sprk-u-ScreenReaderText {
+  position: absolute;
+  left: -10000px;
+  top: auto;
+  width: 1px;
+  height: 1px;
+  overflow: hidden;
+}
+
+.sprk-u-ScreenReaderText.sprk-u-ShowOnFocus:focus {
```

Comment:
The `showOnFocus` class is completely useless without `ScreenReaderText`. Do you think it's necessary to qualify it like this? Basically prevents it from being used without the other, but is that necessary?

Reply:
probably dont need to qualify, somebody might make an override for sprk-u-ScreenReader or something similar and would need ShowOnFocus to be separate.